README.md

```diff
@@ -1,33 +1,40 @@
 Album (code name Aberration)
-
+---------------------------------------
 Copyright (c) 2006-2015 by Matjaž Kovač
 
 Album is a file browsing and tagging utility for BeOS/Haiku. 
 The idea is to have a work pad for pictures from different locations so they may be viewed and tagged. 
 The program is not limited to pictures and BFS (Be File System) volumes, although that makes most sense since some of the functionality has to do with file attributes. 
 
+![Album screenshot](Album.png "Album")
+
 DISTRIBUTION
+------------
 
 This software is distributed under the terms of the MIT license. See LICENSE for details.
 
 
 DOCUMENTATION AND UPDATES
```

Comment:
Since the URL for docs and updates seems dead, that whole section should go.

Reply:
Also DOCUMENTATION AND UPDATES should go?

Reply:
Since the only info on DOCUMENTATION AND UPDATES is the non-working URL, yes.